packages/docs/mint.json

```diff
@@ -0,0 +1,174 @@
+{
+  "$schema": "https://mintlify.com/schema.json",
+  "name": "River",
+  "feedback": {
+    "thumbsRating": true,
+    "suggestEdit": true
+  },
+  "modeToggle": {
+    "default": "dark",
+    "isHidden": true
+  },
+  "logo": {
+    "light": "/logo/light.svg",
+    "dark": "/logo/dark.svg"
+  },
+  "favicon": "/favicon.png",
+  "colors": {
+    "primary": "#8c84f7",
+    "light": "#8c84f7",
+    "dark": "#8c84f7",
+    "anchors": {
+      "from": "#8c84f7",
+      "to": "#8c84f7"
+    },
+    "background": {
+      "light": "#02000A",
+      "dark": "#02000A"
+    }
+  },
+  "topbarLinks": [
+    {
+      "name": "X (Ex. Twitter)",
+      "url": "https://twitter.com/buildonriver"
+    },
+    {
+      "name": "Github",
+      "url": "https://github.com/river-build"
+    }
+  ],
+  "topbarCtaButton": {
+    "name": "Dashboard",
+    "url": "https://dashboard.mintlify.com"
+  },
```

Comment:
can we remove this?

Reply:
i don't know... we can mess with the config in a follow up commit.